actors/abi/sector.go

```diff
@@ -166,10 +166,10 @@ type PartialTicket []byte // 32 bytes
 // TODO Porcu: refactor these types to get rid of the squishy optional fields.
```

Comment:
Please expunge this TODO while you're here.

Reply:
donezo